source/renderer/app/api/ada/index.js

```diff
@@ -250,21 +250,37 @@ export default class AdaApi {
     const { walletId, skip, limit } = request;
 
     const accounts: AdaAccounts = await getAdaWalletAccounts(this.config, { walletId });
-    const accountIndex = accounts[0].index;
-    const page = skip === 0 ? 1 : (skip / limit) + 1;
-    const perPage = limit > 50 ? 50 : limit;
 
     const params = {
-      accountIndex,
-      page,
-      per_page: perPage,
+      accountIndex: accounts[0].index,
+      page: skip === 0 ? 1 : (skip / limit) + 1,
+      per_page: limit > 50 ? 50 : limit,
```

Comment:
Please refactor the magic constant `50` into a proper constant like `TX_PER_PAGE` or similar.

I think `limit` might be better called `maxNumberOfTransactionsToFetch` as i didn't understand the meaning of `limit` in this context at first and was pretty confused before reading through the whole PR.

(in general more speaking var names are preferable to improve readability … the only exception are loop variables)

Reply:
Nice, moving 50 to `TX_PER_PAGE `. 

`limit` and `skip` are part of the V0 scope functions. I kept them as they are used in other methods. `getTransactions` at `api/ada/index.js` is the where they get translated into the V1 variables.